packer/vagrant-box.json

```diff
@@ -1,14 +1,20 @@
 {
     "variables": {
-        "version": "2.0.6",
-        "vm_name": "scf-vagrant-{{isotime \"20060102-1504\"}}",
+        "version": "2.0.6+{{isotime \"20060102-1504\"}}",
```

Comment:
Oh, I think you're right, because the output name is `"output": "scf-{{.Provider}}-v{{user `version`}}.box"`. Let me think about that for a bit.